### DISCUSSION TITLE
Swipe between posts infinitely

### DISCUSSION THREAD

**Discussion by hugburger** [Docile-Alligator/Infinity-For-Reddit#1002] [Completed Ideas] [answered]
The swipe between posts setting has a bug where it doesn't load more posts (ie no infinite scrolling).

To replicate: 1. refresh home or all and open a post
2. Keep swiping left to go to the next post
3. You will reach a point shortly where you cannot open a new post by swiping left

Version 4.3.0 beta 3

**Comment by Docile-Alligator** [comment]
Hi this is not a bug. I just haven't implemented infinite scrolling.

**Comment by hugburger** [comment]
Hi @Docile-Alligator is this likely to be implemented soon or is it low on the backlog?
Thanks!

**Comment by HealYouDown** [comment]
Hey, any updates on this?

Would really love to see this QOL feature implemented as someone that loves swiping between posts.

**Comment by anay-p** [comment]
It would be nice to know if this feature will be implemented soon

**Reply by Wladefant** [reply]
Yes, it will be implemented

**Reply by Wladefant** [reply]
When I don't know

**Reply by anay-p** [reply]
Hopefully soon :)

**Comment by Docile-Alligator** [comment, marked as answer]
Implemented.
Related commits:
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/0a286cd9baab6cf0275d4850f6c2ddb4d2113288
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/be78c20c07ea7a9e724720e675f9078757fa06b9
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/c86408dbe56a27fbf66f06c0d799145437d88fe9
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/fa472a3ad81f752c9568ee8430db0d74764afd53
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/cf3595f22dd35e0bd15d1344c11408f73f64c144
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/0ae9b74729b0ee072bb92dbcfbe3391edabe1c5a
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/93e98b4cb57c2105880a2005c95ba37866fa1e1d
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/fd63dba91b29478009a289323a5d2c2f951dc308
